src/library/Box/propTypes.js

```diff
@@ -8,7 +8,9 @@ export const spacingPropType = oneOfType([
 ]);
 
 export const boxPropTypes = {
+  borderRadius: oneOfType([string, arrayOf(string)]),
   breakpoints: arrayOf(oneOfType([number, string])),
+  color: string,
```

Comment:
"color" usually refers to the text color. Does Mineral have a prop elsewhere that is specifically for background color? I'm assuming either `backgroundColor` or `bgColor` or `bg`.

Reply:
I haven't seen them support color as a prop in any components. They seem to want you to rely on the theme for colors. I can change to backgroundColor to be more explicit about what we are changing.